params/protocol_params.go

```diff
@@ -47,6 +47,9 @@ const (
 	LogDataGas            uint64 = 8     // Per byte in a LOG* operation's data.
 	CallStipend           uint64 = 2300  // Free gas given at beginning of call.
 
+	BlobDAFee      = 2000 // Fee for storing blob to DA provider
+	BlobDAProofGas = 200  // Gas for storing DA proof to span batch
```

Comment:
I would add a comment `It should be average proof size * TxDataNonZeroGasEIP2028`

Reply:
Added.